components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -230,6 +230,17 @@ - (void)updateButtonFont {
   }
   for (MDCFlatButton *button in self.actionButtons) {
     [button setTitleFont:finalButtonFont forState:UIControlStateNormal];
+    [button sizeToFit];
+    CGRect buttonRect = button.frame;
+    buttonRect.size.height = MAX(CGRectGetHeight(buttonRect), MDCDialogActionButtonHeight);
+    buttonRect.size.width = MAX(CGRectGetWidth(buttonRect), MDCDialogActionButtonMinimumWidth);
+    button.frame = buttonRect;
+    CGFloat verticalInset = MIN(0, -(48 - CGRectGetHeight(button.bounds)) / 2);
```

Comment:
Can the second clause be simplified?
(CGRectGetHeight(button.bounds) - 48) / 2

Here an below.